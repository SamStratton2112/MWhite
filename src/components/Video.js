import { Container } from 'react-bootstrap';

const Video = ({ id }) => {

    return (
        <Container style={{width:"50%"}}>
            <div className="ratio ratio-16x9" >
                <iframe src={`https://www.youtube.com/embed/${id}?rel=0`} title="YouTube video" allowFullScreen></iframe>
            </div>
        </Container>
    );
};

export default Video;
