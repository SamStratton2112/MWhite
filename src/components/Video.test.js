import Video from './Video';
import {render} from '@testing-library/react'

describe(Video, ()=>{
    it("renders without crashing", ()=>{
        render(<Video id="test ID" />);
    });
    it("matches snapshot", ()=>{
        const {asFragment} = render(<Video id="test ID" />);
        expect(asFragment()).toMatchSnapshot();
    });
})

  // it("Uses id prop to display embedded youtube video", ()=>{

    // })
