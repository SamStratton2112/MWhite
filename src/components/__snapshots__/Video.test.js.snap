// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="ratio ratio-16x9 "
    style="max-width: 50vw; margin: 0px auto;"
  >
    <iframe
      allowfullscreen=""
      src="https://www.youtube.com/embed/test ID?rel=0"
      title="YouTube video"
    />
  </div>
</DocumentFragment>
`;
